barretenberg/cpp/src/barretenberg/commitment_schemes/gemini/gemini_impl.hpp

```diff
@@ -224,13 +255,39 @@ std::vector<typename GeminiProver_<Curve>::Claim> GeminiProver_<Curve>::compute_
     // A₀₋(X) = F(X) - G(X)/r, s.t. A₀₋(-r) = A₀(-r)
     A_0_neg -= tmp;
 
+    // Reconstruct the batched concatenated polynomial from the batched groups, partially evaluated at r and -r and add
```

Comment:
I wont ask you to un-ascii these comments but keep in mind that we probably are going to need to re-write them eventually

Reply:
Yeah I know, although ascii makes it so much easier to read when looking at code without having doxygen open which is why I've done it